app/models/transcription.rb

```diff
@@ -11,9 +11,25 @@ class Transcription < ApplicationRecord
     in_progress: 1,
     ready: 2, # ready as in "ready for approval"
     unseen: 3
-
   }
 
+  def upload_files_to_storage
+    file_generator = DataExports::TranscriptionFileGenerator.new(self)
+    file_generator.generate_transcription_files.each do |temp_file|
+      # get filename without the temfile's randomly generated unique string
+      basename = File.basename(temp_file)
+      filename = basename.split('-').first + File.extname(basename)
+      files.attach(io: temp_file, filename: filename)
+
+      temp_file.close
+      temp_file.unlink
+    end
+  end
+
+  def remove_files_from_storage
+    files.map(&:purge)
```

Comment:
Not blocking at all but what are these files? i see `has_many_attached :files` but i'm not sure what these actually store, is there a better name than just `transcription.files`?

Reply:
Ah, perhaps `export_files` would be more informative.